migrations/versions/4cb7869866dc_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 4cb7869866dc
+Revises: 8d2b9dcce09
+Create Date: 2015-01-20 10:16:42.409106
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '4cb7869866dc'
+down_revision = '8d2b9dcce09'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('services', sa.Column('service_id', sa.BigInteger(), nullable=True))
```

Comment:
I'm going to squash this change in to the previous commit so that we don't either get an needless extra migration or a commit in the history that breaks the database.